src/common/scripts/utils/renderPage.ts

```diff
@@ -0,0 +1,14 @@
+export default function renderPage(query: string, block: any) {
+    const root: any = document.querySelector(query);
+    root.appendChild(block.getContent());
```

Comment:
document.querySelector может вернуть null
```suggestion
    root?.appendChild(block.getContent());
```